core/src/main/scala/pipeflow/system/PipeFlowSystem.scala

```diff
@@ -0,0 +1,55 @@
+package pipeflow.system
+
+import java.time.LocalDateTime
+
+import akka.actor.ActorSystem
+import com.typesafe.config.{Config, ConfigFactory}
+import org.slf4j.LoggerFactory
+import pipeflow.dsl.nodes.Node
+import pipeflow.system.iso8601.RepeatingInterval
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+import scala.util.{Failure, Success}
+
+
+object PipeFlowSystem {
+  type NodeBuilder = (LocalDateTime) => Node
+
+  private val logger = LoggerFactory.getLogger(PipeFlowSystem.getClass.getName.split("[.$]").last)
```

Comment:
why logger here?

Reply:
By putting it in the companion object, the logger is static, and the same for all instances of the class. Do you think there should be one different logger per instance ? Why ?